### DISCUSSION TITLE
Disable titlebar

### DISCUSSION THREAD

**Discussion by Qaidul250** [Alex313031/thorium#85] [Q&A] [answered]
Is there any way to disable this? My screen resolution isn't that big, and this thing makes my eye feel uncomfortable for some reason.

![Screenshot from 2022-10-20 10-50-54](https://user-images.githubusercontent.com/32721611/196845231-a4060626-0b4e-4cbf-9396-a01018070917.png)

**Comment by midzer** [comment, marked as answer]
Settings -> Appearance -> Use system title bar and borders
